push

```diff
@@ -38,16 +49,14 @@ done
 
 LABEL=${1:-testing}
 
-readonly SOURCE_IMAGE
+if [[ -n "$SOURCE_IMAGE_TAG_OVERRIDE" ]]; then
+    SOURCE_IMAGE_TAG="$SOURCE_IMAGE_TAG_OVERRIDE"
+fi
+
+readonly SOURCE_IMAGE_TAG
 readonly TARGET_IMAGE
 readonly LABEL
 
 set -x
-docker tag "${SOURCE_IMAGE}:latest" "${TARGET_IMAGE}:${LABEL}"
+docker tag "${SOURCE_IMAGE_TAG}" "${TARGET_IMAGE}:${LABEL}"
 gcloud docker -- push "${TARGET_IMAGE}:${LABEL}"
-
-# Only CPU images are made public at this time.
```

Comment:
We stopped updating the `kaggle/python` repo > 1 year  ago. We also don't use this script to retag the image.